cmd/ipfs/dist/install.sh

```diff
@@ -2,20 +2,86 @@
 
 bin=ipfs
 
-# this script is currently brain dead.
-# it merely tries two locations.
-# in the future maybe use value of $PATH.
-
-binpath=/usr/local/bin
-if [ -d "$binpath" ]; then
-  mv "$bin" "$binpath/$bin"
-  echo "installed $binpath/$bin"
-  exit 0
+old_IFS="$IFS"
+IFS=':'
+
+echo "Welcome to the $bin installer."
+echo "With your permission it will try to install the $bin binary into one of"
+echo "the folders found in your \$PATH variable."
+echo
+
+if [ ! -f "$bin" ]; then
+  echo "The '$bin' binary was not found in the current directory."
+  echo "There is nothing to install."
```

Comment:
- a more advanced version could also _test_ the binary is installed, it just needs the right version to _know_. could get the version from the readme.
- if the versions match, the language should still only say `"it seems '$bin $version' is already installed"`. Operative word "seems". we cannot be sure, as some builds will be different but have exact same version.